src/amplifyp/gui/views/input_view.py

```
@@ -632,12 +704,160 @@ def get_all_primers_state(self) -> list[dict[str, Any]]:
+                    "Potential Primer Dimer with quality = "
+                    f"{max_dimer.quality:.1f} "
+                    f"and overlap = {max_dimer.overlap}"
+                )
+                self.info_dimer_text.visible = True
+            else:
+                self.info_dimer_text.value = ""
+                self.info_dimer_text.visible = False
+
+            self.primer_info_panel.visible = True
+
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)